src/EscolaLmsPagesServiceProvider.php

```diff
@@ -26,6 +28,10 @@ public function register()
 
     public function boot()
     {
+        $this->app->bind(
+            ExceptionHandler::class, Handler::class
```

Comment:
I wonder if this wouldn't replace ExceptionHandler for whole App that uses this package?

Reply:
I think it will, I've removed that handler override.